ansible/roles-infra/infra-ibm-upload-plan/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I think this does not need to be another role. I'll merge this in with the template create or template generate.

Reply:
Can we revisit this?
I'm not sure how to handle to call the tf definition upload, plan generation and the sandbox creation itself within the template creation and generate.

Would be possible to rename the role and then create another task file into the role to be imported by the main.yml file?

Reply:
Yes, let us discuss more. I think we will merge a bunch of these together. There are too many shared registered vars where it doesn't make sense to have them as separate roles. If we register a var in role 1 and then need that var in role 2, it either needs to be a default (and we don't know what it will be) or they should be 1 role.